### PR TITLE
Unify iSF_SURFACE_PHYSICS

### LINKED ISSUE #1079: Unify Land Surface Model Index in UPP and UFS-WM
The UFS-WM sets land surface model index ("landsfcmdl") as follows: 1 for NOAH, 2 for NOAH MP, 3 for RUC. This conflicts with the settings in UPP, where 2 for NOAH, 3 for RUC. An update is needed in UPP for processing UFS-based model output.

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
This PR is for unifying  iSF_SURFACE_PHYSICS for land surface model index  in UPP.

### CODE REVIEW

File: sorc/ncep_post.fd/SURFCE.f [hunk 1]
```diff
@@ -6478,7 +6479,8 @@ SUBROUTINE SURFCE
 !     LEAF AREA INDEX
       IF (MODELNAME == 'NCAR'.OR.MODELNAME=='NMM' .OR. &
           MODELNAME == 'FV3R' .OR. MODELNAME=='RAPR')THEN
-      IF (iSF_SURFACE_PHYSICS == 2 .OR. MODELNAME=='FV3R' .OR. MODELNAME=='RAPR') THEN
+      IF (iSF_SURFACE_PHYSICS == 2 .OR. iSF_SURFACE_PHYSICS == 1 .OR. &
+          MODELNAME=='FV3R' .OR. MODELNAME=='RAPR') THEN
```
Comment (HuiyaChuang-NOAA) [inline sorc/ncep_post.fd/SURFCE.f]: @WenMeng-NOAA 
It seems line 6481 already has if condition for 'FV3R' .or. 'RAPR'?
I know this was not added by you but just seems redundant

File: sorc/ncep_post.fd/WRFPOST.F [hunk 2]
```diff
@@ -423,11 +424,12 @@ PROGRAM WRFPOST
...
             print*,'landsfcmdl not found; assigning to 2'
-            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
-          endif
-          if(iSF_SURFACE_PHYSICS<2)then
-            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
+            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH MP
+            ! LSM physics 1 for NOAH
           endif
+!          if(iSF_SURFACE_PHYSICS<2)then
+!            iSF_SURFACE_PHYSICS=2 !set LSM physics to 2 for NOAH
```
Comment (HuiyaChuang-NOAA) [inline sorc/ncep_post.fd/WRFPOST.F:431]: I think this block supports GFS, FV3R, and RAP/HRRR, right?  I believe RAP/HRRR still uses RUC.

File: sorc/ncep_post.fd/SURFCE.f (hunk 1 above)
Comment (WenMeng-NOAA) [reply sorc/ncep_post.fd/SURFCE.f]: @HuiyaChuang-NOAA Thanks for catching that. I will fix it.

File: sorc/ncep_post.fd/WRFPOST.F (hunk 2 above)
Comment (WenMeng-NOAA) [reply sorc/ncep_post.fd/WRFPOST.F:431]: @HuiyaChuang-NOAA That block is reading model attribute 'landsfcmdl' from UFS-based model output.  The RAP/HRRR process is in another block (line 395).

Comment (HuiyaChuang-NOAA) [reply sorc/ncep_post.fd/WRFPOST.F:431]: @WenMeng-NOAA thank you for explanation.  I will approve next

**Comment by WenMeng-NOAA** [comment]
@HuiyaChuang-NOAA The UPP RTs have been completed on WCOSS2 with no baseline changes.

**Comment by WenMeng-NOAA** [comment]
@gspetro-NOAA You may start the UPP RTs on R&D machines. There should be no baseline changes with this PR.

**Review by gspetro-NOAA — APPROVED** [review]
RTs passed on Hera, Orion, and Hercules with no baseline changes.

**Review by HuiyaChuang-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
This PR is ready for merging.

**Pull request merged**